code/modules/mod/mod_types.dm

```diff
@@ -224,6 +224,7 @@
 		/obj/item/mod/module/dna_lock/reinforced,
 		/obj/item/mod/module/emp_shield/pulse,
 		/obj/item/mod/module/status_readout,
+		/obj/item/mod/module/storage,
```

Comment:
tiny thing but could you move this and status readout up in the list? passive items should be first

Reply:
yeah sure